sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionReceiptQuery.java

```diff
@@ -204,7 +204,8 @@ ExecutionState getExecutionState(Status status, Response response) {
                 break;
 
             default:
-                return ExecutionState.REQUEST_ERROR;
+//                return ExecutionState.REQUEST_ERROR;
+                return super.getExecutionState(status, response);
```

Comment:
We have the same logic in `TransactionRecordQuery.java`, shall we change it there too to handle `PlatformNotActive`

Reply:
I think yes, we just need to sync to discuss the approach